fhir-persistence-schema/docs/SchemaToolUsageGuide.md

```diff
@@ -522,14 +522,43 @@ Note: the jar file is stored locally in `fhir-persistence-schema/target` or in t
     - stored-procedures.sql:  `db2 -td@ -vf stored-procedures.sql`
 
 
+----------------------------------------------------------------
 # V0021 - Drops the DOMAINRESOURCE and RESOURCE tables
 
 If there is data in the DOMAINRESOURCE and RESOURCE table groups, which is unexpected, the administrator may run the tool with `--force-unused-table-removal` to force the removal of the unused tables.
 
+----------------------------------------------------------------
+# Database Size Report (Db2, PostgreSQL)
 
+Run this command to show a summary of the space used by IBM FHIR Server resources and their related schema objects:
 
-# List of IBM FHIR Server Persistence Schema Tool Flags
+``` shell
+java -jar ./fhir-persistence-schema-${VERSION}-cli.jar \
+--prop-file fhiradmin.properties \
+--db-type postgresql \
+--schema-name FHIRDATA \
+--show-db-size
+```
 
+To include per-table and per-index in the output, add the `--show-db-size-detail` flag:
+
+``` shell
+java -jar ./fhir-persistence-schema-${VERSION}-cli.jar \
+--prop-file fhiradmin.properties \
+--db-type postgresql \
+--schema-name FHIRDATA \
+--show-db-size \
+--show-db-size-detail
+```
+
+The detail rows are tab-separated, making it easy to load the data into a spreadsheet for further analysis.
+
+**Notes:**
+1. The size report is only supported on Db2 and PostgreSQL databases.
+2. The size report is intended as a guide to understand the relative space distribution of objects in the IBM FHIR Server data schema. The report is not intended to replace database utilities for calculating the total size of the database.
+
+----------------------------------------------------------------
+# List of IBM FHIR Server Persistence Schema Tool Flags
```

Comment:
should we add the `--tenant-name` flag to this list as well?

Reply:
good point. done.